test/integration/targets/nxos_static_routes/tests/cli/overridden.yml

```diff
@@ -0,0 +1,64 @@
+---
+  - debug:
+      msg: "Start nxos_static_routes overridden tests connection={{ ansible_connection }}"
+
+  - include_tasks: populate_config.yaml
+
+  - block:
+      - name: Gather static_routes facts
+        nxos_facts: &facts
+          gather_subset:
+            - '!all'
+            - '!min'
+          gather_network_resources: static_routes
+
+      - name: Overridden
```

Comment:
Again here, please add a test to update an existing route property or multiple properties with state `overridden` to make sure it gets processed properly.